clio-client/src/main/scala/org/broadinstitute/clio/client/util/IoUtil.scala

```diff
@@ -35,11 +35,22 @@ trait IoUtil {
   }
 
   def deleteDirectoryRecursively(directory: Path): Unit = {
-    Files
-      .walk(directory, FileVisitOption.FOLLOW_LINKS)
-      .sorted(Comparator.reverseOrder())
-      .map[File](path => path.toFile)
-      .forEach(file => file.deleteOnExit())
+    val _ =
+      Files.walkFileTree(directory, new SimpleFileVisitor[Path]() {
```

Comment:
@pshapiro4broad I haven't used `FileVisitor` before, does this look sane?

Reply:
Looks good to me.